### ISSUE TITLE
Critical Error when searching on plugins page: Uncaught Error: Call to a member function get_column_count() on null in [...]/plugins/woocommerce/src/Packages.php:260

### ISSUE THREAD

**Issue by EdithAllison** [woothemes/woocommerce#53658]
### Prerequisites

- [x] I have carried out troubleshooting steps and I believe I have found a bug.
- [x] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

Searching on the plugins page results in a critical error. Search does not work, load icon keeps spinning. 

### Expected behavior

I expect search to bring up a list of matching plugins

### Actual behavior

`
Uncaught Error: Call to a member function get_column_count() on null in .../wp-content/plugins/woocommerce/src/Packages.php:260  
{
    "error": {
        "type": 1,
        "file": ".../plugins/woocommerce/src/Packages.php",
        "line": 260
    },
    "remote-logging": true,
    "backtrace": [
        "",
        "#0.../wp-includes/class-wp-hook.php(326): Automattic\WooCommerce\Packages::display_notice_for_merged_plugins('woocommerce-ord...')",
        "#1 .../wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)",
        "#2 .../wp-includes/plugin.php(517): WP_Hook->do_action(Array)",
        "#3 /.../wp-admin/includes/class-wp-plugins-list-table.php(1491): do_action('after_plugin_ro...', 'woocommerce-ord...', Array, 'search')",
        "#4 .../wp-admin/includes/class-wp-plugins-list-table.php(711): WP_Plugins_List_Table->single_row(Array)",
        "#5 .../wp-admin/includes/class-wp-list-table.php(1712): WP_Plugins_List_Table->display_rows()",
        "#6 .../wp-admin/includes/class-wp-list-table.php(1639): WP_List_Table->display_rows_or_placeholder()",
        "#7 .../wp-admin/includes/ajax-actions.php(4834): WP_List_Table->display()",
        "#8 .../wp-includes/class-wp-hook.php(324): wp_ajax_search_plugins('')",
        "#9 .../wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)",
        "#10 .../wp-includes/plugin.php(517): WP_Hook->do_action(Array)",
        "#11 .../wp-admin/admin-ajax.php(192): do_action('wp_ajax_search-...')",
        "#12 {main}",
        "thrown"
    ]
}`

### Steps to reproduce

1. Go to Plugins page
2. Search 

### WordPress Environment

<details><summary>System Status Report</summary>

```
### WordPress Environment ###

WordPress address (URL): [Redacted]
Site address (URL): [Redacted]
WC Version: 9.4.3
Legacy REST API Package Version: ✔ 1.0.4
Action Scheduler Version: ✔ 3.8.2
Log Directory Writable: ✔
WP Version: 6.7.1
WP Multisite: –
WP Memory Limit: 768 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_US
External object cache: ✔

### Server Environment ###

Server Info: Apache
PHP Version: 8.2.26
PHP Post Max Size: 256 MB
PHP Time Limit: 120
PHP Max Input Vars: 3000
cURL Version: 8.2.1
OpenSSL/3.1.4

SUHOSIN Installed: –
MySQL Version: 8.0.39-30
Max Upload Size: 256 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

[REDACTED]

### Post Type Counts ###

acf-field: 20
acf-field-group: 8
attachment: 5840
aw_workflow: 10
blocks: 1
boxzilla-box: 3
create-portfolio: 1
custom_css: 1
customize_changeset: 26
faq: 1
flamingo_contact: 20
gd_place: 11
gift_card: 2
hrf_faq: 23
ig_campaign: 3
ig_message: 10
mc4wp-form: 1
nav_menu_item: 91
oembed_cache: 23
oxy-testimonial: 3
page: 31
post: 3
prl_engine: 4
prl_hook: 1
product: 4889
product_variation: 397
revision: 181
seedprod: 3
shipping_zone: 9
shop_coupon: 23
shop_order: 47836
shop_order_refund: 8823
shop_subscription: 845
sidebar: 1
wafs: 2
was: 11
wc_order_status: 8
wc_pickup_location: 1
wc_product_tab: 1
wcp_ruleset: 1
wpcf7_contact_form: 2
wpephpcompat_jobs: 32
yith-wcbm-badge: 6

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (79) ###

Admin Columns Pro: by AdminColumns.com – 6.4.13
Admin Menu Editor: by Janis Elsts – 1.12.4
Advanced Custom Fields: by WP Engine – 6.3.11
AJAX Login and Registration modal popup - PRO: by Maxim K – 2.18
FiboSearch - AJAX Search for WooCommerce (Pro): by FiboSearch Team – 1.29.0
AutomateWoo: by WooCommerce – 6.1.3
Basket Packing List: by BLSSOL – 1.0
Boxzilla - Google Analytics: by ibericode – 1.1.9
Boxzilla - MailChimp integration: by ibericode – 1.1.3
Boxzilla - Theme Pack: by ibericode – 1.1.5
Boxzilla: by ibericode – 3.3.3
PublishPress Capabilities: by PublishPress – 2.16.0
Classic Editor: by WordPress Contributors – 1.6.7
Code Snippets: by Code Snippets Pro – 3.6.6.1
Conditional Payments for WooCommerce: by Lauri Karisola / WP Trio – 3.2.0
Contact Form 7: by Takayuki Miyoshi – 6.0.1
Customer Delivery Route Logs Plugin: by BLSSOL – 1.2
Favicon by RealFaviconGenerator: by Philippe Bernard – 1.3.37
Filter Orders by User meta: by Mahboob Ur Rehman – 1.1
FluentSMTP: by FluentSMTP & WPManageNinja Team – 2.2.83
Fluent Support Pro: by WPManageNinja LLC – 1.8.4
Fluent Support: by WPManageNinja LLC – 1.8.4
Site Kit by Google: by Google – 1.141.0
Grow Development Subscription Cancellation Flow: by Grow Development – 1.0.1
WooCommerce Subscriptions Dashboard: by Grow Development – 1.1.2
Tag Pilot FREE - Google Tag Manager Integration for WooCommerce: by Tag Concierge – 1.12.3
Jetpack Protect: by Automattic - Jetpack Security team – 3.1.1
Jetpack: by Automattic – 14.1
JSM Show Post Metadata: by JS Morisset – 4.6.1
Kadence WooCommerce Email Designer: by Kadence WP – 1.5.13
MC4WP: Mailchimp for WordPress: by ibericode – 4.9.19
MC4WP: Mailchimp for WordPress Premium: by ibericode – 4.10.6
Metorik Helper: by Metorik – 2.0.8
My Preferences: by David Kyalo – 1.0
WooCommerce Sync for QuickBooks Desktop - by MyWorks Software: by MyWorks Software – 1.8.3
Ottorg Subscription Lookup: by David Towoju – 0.2
Ottorg — Subscription Vacation Manager: by Helder Vilela – 1.8.3
Ottorg — Custom User Profile: by Helder Vilela – 1.9.1
Database Manager - WP Adminer: by Pexle Chris – 3.0.3.1
Product Recommendations - Custom Locations: by WooCommerce – 2.0.4
SeedProd Pro: by SeedProd – 6.15.13.1
Speed Optimizer: by SiteGround – 7.7.1
SiteGround Migrator: by SiteGround – 2.0.9
Sucuri Security - Auditing, Malware Scanner and Hardening: by Sucuri Inc. – 1.9.6
SVG Support: by Benbodhi – 2.5.8
Category Order and Taxonomy Terms Order: by Nsp-Code – 1.8.7
Toolbox for WooCommerce Subscriptions: by Shop Plugins – 1.5.8
User Switching: by John Blackbourn – 1.9.1
Weekly Baskets: by David Kyalo – 1.0.1
Do We Deliver: by ottawaorganics.ca – 1.0
WooCommerce Advanced Quantity: by Morning Train Technologies ApS – 3.0.6
Woo Import Export: by VJInfotech – 5.9.27
WooCommerce.com Update Manager: by Automattic – 1.0.3
WooCommerce Advanced Bulk Edit: by George Iron & Yas G. – 5.4.4.7.1
Woocommerce Advanced Shipping Zones: by Jeroen Sormani – 1.0.1
WooCommerce Advanced Shipping: by Jeroen Sormani – 1.1.4
WooCommerce Cost of Goods: by SkyVerge – 2.13.3
BLAZING Email Transfer Payment Gateway: by Massoud Shakeri
BlazingSpider – 2.6.0

WooCommerce Bambora Gateway: by SkyVerge – 2.9.3
WooCommerce Gift Cards: by Woo – 2.1.1
WooCommerce Legacy REST API: by WooCommerce – 1.0.4
WooCommerce Name Your Price: by Kathy Darling – 3.5.14
Advanced Order Export For WooCommerce (Pro): by AlgolPlus – 3.5.6
WooCommerce Order Status Manager: by SkyVerge – 1.15.4
WooCommerce Print Invoices/Packing Lists: by SkyVerge – 3.13.7
WooCommerce Product Bundles: by Woo – 8.2.0
WooCommerce Product Recommendations: by Woo – 4.1.1
HUSKY - Products Filter Professional for WooCommerce: by realmag777 – 3.3.6.4
Store Toolkit for WooCommerce: by Visser Labs – 2.4.1
WooCommerce Subscriptions CSV Importer and Exporter: by Prospress Inc – 2.1.0
WooCommerce Subscriptions - Preserve Billing Schedule: by Prospress Inc. – 1.0.0
WooCommerce Subscriptions: by WooCommerce – 6.9.1
WooCommerce: by Automattic – 9.4.3
Wordfence Security: by Wordfence – 8.0.1
Yoast SEO: by Team Yoast – 24.0
WP Crontrol: by John Blackbourn – 1.17.1
Refer a Friend for WooCommerce PREMIUM: by WPGens – 4.1.4
WPS Hide Login: by WPServeur
NicolasKulka
wpformation – 1.9.17.1

YITH WooCommerce Wishlist: by YITH – 4.0.1

### Inactive Plugins (10) ###

Advanced Local Pickup PRO: by zorem – 1.7.2
AJAX Login and Registration modal popup - PRO: by Maxim K – 2.14
WP-Optimize - Clean, Compress, Cache: by TeamUpdraft
DavidAnderson – 3.7.1

WP All Export - WooCommerce Export Add-On Pro: by Soflyy – 1.0.9
WP All Export Pro: by Soflyy – 1.8.5
WP All Import - WooCommerce Import Add-On Pro: by Soflyy – 4.0.0
WP All Import Pro: by Soflyy – 4.8.5
WP Mail Logging: by WP Mail Logging Team – 1.13.1
WP Rocket: by WP Media – 3.17.1
WP Rocket | Change Parameters: by WP Rocket Support Team –

### Dropin Plugins () ###

advanced-cache.php: advanced-cache.php
object-cache.php: Memcached

### Must Use Plugins (2) ###

pexlechris_adminer_avoid_conflicts_with_other_plugins.php: by  – 3.0.3
WP STAGING Optimizer: by WP STAGING – 1.5.9

### Settings ###

Legacy API Enabled: ✔
Force SSL: ✔
Currency: CAD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: bundle (bundle)
external (external)
gift-card (gift-card)
grouped (grouped)
simple (simple)
subscription (subscription)
variable (variable)
variable subscription (variable-subscription)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: ✔
Enforce Approved Product Download Directories: –
HPOS feature enabled: –
Order datastore: WC_Order_Data_Store_CPT
HPOS data sync enabled: –

### Logging ###

Enabled: ✔
Handler: Automattic\WooCommerce\Internal\Admin\Logging\LogHandlerFileV2
Retention period: 30 days
Level threshold: –
Log directory size: 22 MB

### WC Pages ###

Shop base: #6 - /shop/
Cart: #7 - /cart/ -  Contains the [woocommerce_cart] shortcode
Checkout: #8 - /checkout/ -  Contains the [woocommerce_checkout] shortcode
My account: #9 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Office Organics ( Flatsome Child )
Version: 1.9.19
Author URL: https://heldervilela.com
Child Theme: ✔
Parent Theme Name: Flatsome
Parent Theme Version: 3.19.7
Parent Theme Author URL: https://uxthemes.com
WooCommerce Support: ✔

### Templates ###

Overrides: flatsome/woocommerce/archive-product.php
flatsome/woocommerce/cart/cart-empty.php
flatsome/woocommerce/cart/cart-shipping.php
flatsome-child/woocommerce/cart/cart.php
flatsome/woocommerce/cart/mini-cart.php
flatsome/woocommerce/checkout/form-checkout.php
flatsome/woocommerce/checkout/form-coupon.php
flatsome/woocommerce/checkout/order-received.php
flatsome/woocommerce/checkout/terms.php
flatsome/woocommerce/checkout/thankyou.php
flatsome/woocommerce/content-product-cat.php
flatsome/woocommerce/content-product.php
flatsome/woocommerce/content-single-product.php
flatsome/woocommerce/content-widget-product.php
flatsome/woocommerce/global/breadcrumb.php
flatsome/woocommerce/global/quantity-input.php
flatsome/woocommerce/global/wrapper-end.php
flatsome/woocommerce/global/wrapper-start.php
flatsome/woocommerce/loop/header.php
flatsome/woocommerce/loop/loop-end.php
flatsome/woocommerce/loop/loop-start.php
flatsome/woocommerce/loop/pagination.php
flatsome/woocommerce/loop/rating.php
flatsome/woocommerce/loop/result-count.php
flatsome/woocommerce/loop/sale-flash.php
/home/customer/www/ottawaorganics.ca/public_html/wp-content/plugins/ajax-login-and-registration-modal-popup-pro/templates/wc-form.php version 3.6.0 is out of date. The core version is 9.2.0
flatsome/woocommerce/myaccount/navigation.php
flatsome/woocommerce/notices/error.php
flatsome/woocommerce/notices/notice.php
flatsome/woocommerce/notices/success.php
flatsome/woocommerce/product-searchform.php
flatsome-child/woocommerce/single-product/add-to-cart/simple.php
flatsome-child/woocommerce/single-product/add-to-cart/variation-add-to-cart-button.php
flatsome/woocommerce/single-product/photoswipe.php
flatsome/woocommerce/single-product/price.php
flatsome/woocommerce/single-product/product-image.php
flatsome/woocommerce/single-product/product-thumbnails.php
flatsome/woocommerce/single-product/rating.php
flatsome/woocommerce/single-product/related.php
flatsome/woocommerce/single-product/review.php
flatsome/woocommerce/single-product/sale-flash.php
flatsome/woocommerce/single-product/share.php
flatsome/woocommerce/single-product/short-description.php
flatsome/woocommerce/single-product/tabs/tabs.php
flatsome/woocommerce/single-product/title.php
flatsome/woocommerce/single-product/up-sells.php
flatsome/woocommerce/single-product-reviews.php
flatsome/woocommerce/single-product.php

Outdated Templates: ❌
					
					
						Learn how to update					 |
					
						
					
					
						Clear system status theme info cache

### Templates ###

thumbnail: 
medium: 
medium_large: 
large: 
1536x1536: 
2048x2048: 
theme_product_thumbnail: 
woocommerce_thumbnail: 
woocommerce_single: 
woocommerce_gallery_thumbnail: 
wc_order_status_icon: 
dgwt-wcas-product-suggestion: 

### Subscriptions ###

WCS_DEBUG: ✔ No
Subscriptions Mode: ✔ Live
Subscriptions Live URL: https://ottawaorganics.ca
Subscriptions-core Library Version: 7.7.2
Subscriptions Template Theme Overrides: flatsome-child/woocommerce/cart/cart-recurring-shipping.php
Subscription Statuses: wc-active: 180
wc-expired: 1
wc-pending: 1
wc-on-hold: 22
wc-cancelled: 641

WooCommerce Account Connected: ✔ Yes
Active Product Key: ❌ No
Custom Retry Rules: ✔ No
Custom Retry Rule Class: ✔ No
Custom Raw Retry Rule: ✔ No
Custom Retry Rule: ✔ No
Retries Migration Status: ✔ Completed
Report Cache Enabled: ✔ Yes
Cache Update Failures: ✔ 0 failure

### Store Setup ###

Country / State: Canada — Ontario

### Subscriptions by Payment Gateway ###

Bambora Credit Cards: wc-cancelled: 571
wc-active: 174
wc-on-hold: 10
wc-expired: 1
wc-pending: 1

other: wc-on-hold: 8
wc-cancelled: 47
wc-active: 2

BLAZING Email Transfer: wc-on-hold: 4
wc-cancelled: 16
wc-active: 3


### Payment Gateway Support ###

Bambora Credit Cards: products
card_types
payment_form
charge
charge-virtual
authorization
capture_charge
partial_capture
customer_decline_messages
refunds
voids
tokenization
customer_id
add_payment_method
token_editor
subscriptions
subscription_suspension
subscription_cancellation
subscription_reactivation
subscription_amount_changes
subscription_date_changes
multiple_subscriptions
subscription_payment_method_change_customer
subscription_payment_method_change_admin

Cash on delivery: products
BLAZING Email Transfer: products

### Gift Cards ###

Database Version: 2.1.1
Loopback Test: ✔
Task Queueing Test: ✔

### Product Bundles ###

Database Version: 8.2.0
Loopback Test: ✔
Template Overrides: –

### Bambora Credit Cards ###

Environment: Production
Tokenization Enabled: ✔
Debug Mode: Off

### Name Your Price ###

Version: 
Database Version: 
Template Overrides: –

### Admin ###

Enabled Features: activity-panels
analytics
product-block-editor
coupons
core-profiler
customize-store
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
mobile-app-banner
navigation
onboarding
onboarding-tasks
pattern-toolkit-full-composability
product-custom-fields
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
printful
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page
launch-your-store

Disabled Features: product-data-views
experimental-blocks
minified-js
product-pre-publish-modal
settings
async-product-editor-category-field
product-editor-template-system
blueprint
reactify-classic-payments-settings
use-wp-horizon

Daily Cron: ✔ Next scheduled: 2024-12-13 02:49:45 -05:00
Options: ✔
Notes: 223
Onboarding: completed

### Action Scheduler ###

Complete: 73,608
Oldest: 2024-11-11 04:11:47 -0500
Newest: 2024-12-12 04:10:42 -0500

Failed: 8,403
Oldest: 2021-03-10 20:09:22 -0500
Newest: 2024-12-09 06:52:11 -0500

Pending: 201
Oldest: 2024-12-12 04:11:42 -0500
Newest: 2025-01-07 04:00:10 -0500


### Product Recommendations ###

Database Version: 4.1.1
Loopback Test: ✔
Task Queueing Test: ✔
Page Cache Test: Cache in use

### Status report information ###

Generated at: 2024-12-12 04:11:18 -05:00
```
</details>

### Isolating the problem

- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/products/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Comment by EdithAllison** [comment]
I was able to consistently reproduce this bug on two other sites running the latest version of WooCommerce (v9.4.3)

**Comment by barryhughes** [comment]
Hi @EdithAllison. This would seem to be a duplicate of https://github.com/woocommerce/woocommerce/issues/53057, which was fixed in https://github.com/woocommerce/woocommerce/pull/53438. I'd recommend holding tight until 9.5.1 is released, which should be tomorrow or the next day.
